src/helpers/game_utils.ts

```diff
@@ -29,7 +29,7 @@ export async function playCorrectGuessSong(gameSession: GameSession) {
     });
 }
 
-async function getFilteredSongList(guildPreference: GuildPreference): Promise<{ songs: QueriedSong[], countBeforeLimit: number }> {
+async function getFilteredSongList(guildPreference: GuildPreference, lastPlayedSongs?: Array<string>): Promise<{ songs: QueriedSong[], countBeforeLimit: number }> {
```

Comment:
Rename `lastPlayedSongs` to something more generic like `ignoredVideoIds` only for this method